readme.md

```diff
@@ -218,7 +221,9 @@
 - [Admin User Chatting](https://github.com/hbakouane/admin-user-chatting) - A Live chat between users and admins made with Livewire.
 - [Laravel, Vue & Admin LTE Boilerplate](https://github.com/hbakouane/adminlte-laravel-vue) - Laravel, Vue and Admin LTE integrated in one project.
 - [PHPFlasher](https://github.com/php-flasher/php-flasher) - A powerful and flexible flash notification system for PHP
-With Symfony and Laravel implementations. by [@yoeunes](https://github.com/yoeunes)
+  With Symfony and Laravel implementations. by [@yoeunes](https://github.com/yoeunes)
+- [Block Content Selected](https://www.drupal.org/project/bcs) - Inject and render any types of nodes to any block using autocomplete with view mode. By [@kamalkech](https://github.com/kamalkech)
+- [Node Layout Builder](https://www.drupal.org/project/node_layout_builder) - Custom layout of page node. By [@kamalkech](https://github.com/kamalkech)
```

Comment:
If you could change this to Github repositories, better cause we can't identify it

Reply:
@yezz123 drupal use by default gitlab (commercial politic
) to deploy and publish all modules for drupal x

Reply:
what I understand is this deployed directly without sharing the repositories code?

Reply:
@yezz123 if you check details page of each item you will see all information about that
![Node Layout buil](https://user-images.githubusercontent.com/1313428/144402040-da847eee-4a61-40fe-8ac7-20bfb76d1ffa.png)

Reply:
Yeah, I see, that's Great the issue right now is that we are working on a website that fetches the Moroccan data, so we will work on some exceptions like drupal and Gitlab.

cc @yjose

Reply:
@JefferyHus can you read my first comment, and what s problem if any solution publish on another instead of github ?

Reply:
I have read the comment before I write mine. The reason we would love to keep OSS projects under github (for now) is because we are planning to turn this into a website and here we are going to use github API to fetch more details about projects, which we can't do for a drupal project like the one you shared. I still believe it will be fine to share it on github? Otherwise, we can raise a question about how OSS is your project

Reply:
@JefferyHus understood but keep in your mind this is a process / requirement of drupal to publish packages on gitlab usinc account drupal.org so your proposition is not ok for this case, for more details you can check Drupal doc about publish custom module.